scripts/package.js

```diff
@@ -120,34 +120,68 @@ export default class Package extends EventEmitter {
     }
   }
 
-  async build(options, outputOptions) {
+  async build(options, _watch = false) {
     this.emit('build:before')
 
+    // Add build suiffix if needed
     if (this.options.buildSuffix) {
       this.convertTo(this.options.buildSuffix)
     }
 
-    this.logger.info('Building')
-
     // https://rollupjs.org/guide/en#javascript-api
-    const bundle = await rollup(rollupConfig({
+
+    const config = rollupConfig({
       rootDir: this.options.rootDir,
       ...options
-    }, this.pkg))
+    }, this.pkg)
+
+    if (_watch) {
+      // Watch
+      const watcher = watch(config)
+      watcher.on('event', (event) => {
+        switch (event.code) {
+          // The watcher is (re)starting
+          case 'START': return this.logger.debug('Watching for changes')
```

Comment:
Could we "reference" the event codes somehow? It's unlikely that they'll break but it'd be *safer* to have a Map or a lookup object 🤔